extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -295,7 +295,15 @@ export function maybeResizeTextarea(element) {
         // Prevent the textarea from shrinking if it has not yet expanded.
         const hasExpanded =
           AMP_FORM_TEXTAREA_HAS_EXPANDED_DATA in element.dataset;
-        const shouldResize = hasExpanded || scrollHeight <= minScrollHeight;
+
+        // There is super specific a bug in Chrome affecting scrollHeight calculation
+        // for textareas with padding when the document is zoomed in.
+        // It makes the scrollHeight calculation off by ~1px.
+        // This is why we have a small error margin.
+        // TODO: Remove error margin when chrome bug is resolved (https://bugs.chromium.org/p/chromium/issues/detail?id=1171989).
+        const errorMargin = /google/i.test(win.navigator.vendor) ? 3 : 0;
```

Comment:
Is `win` preferred to `this.win_` here? And should we also be checking if the document is zoomed at all?

Reply:
1. We don't have access to this.win and it isn't a property of the textarea element either (not an AMP Component).  win is already calculated earlier in the function, so it is better than writing element.ownerDocument.defaultView again.
2. tabs.getZoom() would returns a Promise and I think would do the trick. My fear is that zoom isn't the root cause and it is really related to device pixel ratio - zoom is just the easiest repro method. Due to not understanding the underlying Chrome bug well enough, I'd rather stay on the safe side! WDYT

Reply:
1. Oops 😬  You're so right, thanks for clarifying. 
2. It should be fine to keep. I don't foresee anything harmful happening because in the case where the bug is not present, and we are "overeager" to resize, the error margin doesn't actually factor into the given height.